src/coordinator/hybrid_logical_clock.hpp

```diff
@@ -31,6 +34,15 @@ struct Hlc {
   bool operator==(const uint64_t other) const { return logical_id == other; }
   bool operator<(const uint64_t other) const { return logical_id < other; }
   bool operator>=(const uint64_t other) const { return logical_id >= other; }
+
+  friend std::ostream &operator<<(std::ostream &in, const Hlc &hlc) {
+    auto wall_clock = std::chrono::system_clock::to_time_t(hlc.coordinator_wall_clock);
+    in << "Hlc { logical_id: " << (int)hlc.logical_id;
```

Comment:
I started to think I am missing something here with the casts to `int`, so please enlighten me!